python/tester/data/LATISS.yaml

```diff
@@ -0,0 +1,4 @@
+query:
+  collections: LATISS/raw/all
+  where: "instrument='LATISS' and exposure.science_program='AUXTEL_PHOTO_IMAGING' and exposure.day_obs=20240403"
```

Comment:
I think of this as a convenient filter so the number of selected visits is relatively manageable. In practice we'll probably often modify it depending on the goals of the tests.

Reply:
I'm not entirely sure about calib collisions.  I think we should be okay mixing data from various original dates.  Some data may not be processed with the "best" calib suitable for it, but it'd be more of a pipeline quality issue, and it doesn't really matter as
tests for prompt service, I think.

Reply:
Well, what happened with HSC was that preload would fail catastrophically if it tried to process visits from different (original) dates on the same (test) date. The only workaround we found was to erase calibs after every run...

Reply:
Thank you for reminding me about the calib collision issue. 

If we mock the observing date (which was necessary before [DM-43495](https://rubinobs.atlassian.net/browse/DM-43495)) to be the test date, processing all use calibration for the test date, hence no calib collisions. 
If we don't mock the observing date, calib collisions can happen just like HSC. See the other thread re header modification.